README.md

```diff
@@ -35,6 +35,13 @@ spec:
           grpc:
           http:
     processors:
+      memory_limiter:
+        check_interval: 1s
+        limit_mib: 500
+        spike_limit_mib: 100
+      batch:
```

Comment:
Why 1000 and 2s? The default values are different https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/batchprocessor/README.md